browser/src/ClinvarVariantsTrack/ClinvarVariantPropType.ts

```diff
@@ -1,6 +1,33 @@
 import PropTypes from 'prop-types'
 
-const ClinvarVariantPropType = PropTypes.shape({
+type ClinvarVariantPropType = {
+  clinical_significance: string
+  clinvar_variation_id: string
+  gnomad?: {
+    exome?: {
+      ac: number
+      an: number
+      filters: string[]
+    }
+    genome?: {
+      ac: number
+      an: number
+      filters: string[]
+    }
+  }
+  gold_stars: number
+  hgvsc?: string
+  hgvsp?: string
+  in_gnomad: boolean
+  major_consequence?: string
+  pos: number
+  review_status: string
+  transcript_id: string
+  variant_id: string
+}
+
+// @ts-expect-error TS(2322) FIXME: Type 'Requireable<InferProps<{ clinical_significan... Remove this comment to see the full error message
+const ClinvarVariantPropType: PropTypes.Requireable<ClinvarVariantPropType> = PropTypes.shape({
```

Comment:
Sometimes PropTypes are stripped out, sometimes they are not... wonder how this is determined.